File: solutions/r/darts/1/darts.R
score <- function(x, y) {
  distance = sqrt(x^2 + y^2)
  if (distance > 10) { 0 }
  else if (distance > 5) { 1 }
  else if (distance > 1) { 5 }
  else { 10 }
}

File: solutions/python/darts/1/darts.py
from math import sqrt, inf

SCORES = {
    1: 10,
    5: 5,
    10: 1,
    inf: 0
}
def score(x, y):
    distance = sqrt(x**2 + y**2)
    for radius, points in SCORES.items():
        if distance <= radius:
            return points
